app-creator/client/src/widgets/palette-picker/test/palette-picker_test.ts

```diff
@@ -0,0 +1,22 @@
+/**
+ *  @fileoverview This file tests the palette-picker widget.
```

Comment:
Let's add a test simulating a palette selection and that the event fires correctly and with the right data!